wger/core/templates/user/login.html

```diff
@@ -14,6 +14,29 @@
     {% trans 'Login' as submit_text %}
     {% render_form_fields form submit_text %}
 </form>
+<div class="row">
```

Comment:
Instead on line styling, why wouldn't you add them in a css file?

Reply:
I preferred  line styling because when I style in the css file.The changes do not reflected